src/pages/index.js

```diff
@@ -84,19 +181,60 @@ function openEditForm() {
 
 editPopup.setEventListeners();
 
-variables.profileEditButtonSelector.addEventListener("click", () => {
+variables.profileEditButton.addEventListener("click", () => {
   openEditForm();
 });
 
-variables.profileAddButtonSelector.addEventListener("click", () =>
-  openAddForm()
+// POPUP WITH FORM - AVATAR FORM
+
+const avatarModal = new PopupWithForm(
+  variables.avatarModal,
+  handleAvatarFormSubmit
 );
+avatarModal.setEventListeners();
+
+function openAvatarForm() {
+  avatarModal.open();
+}
+
+variables.avatarIcon.addEventListener("click", () => {
+  openAvatarForm();
+});
+
+function handleAvatarFormSubmit(inputValues) {
+  setButtonText(variables.avatarModalButton, "Saving...");
+  api
+    .updateAvatar(inputValues.link)
+    .then((res) => {
+      variables.avatarImage.src = inputValues.link;
```

Comment:
`      variables.avatarImage.src = inputValues.link;
`

Class `UserInfo` should set the entire user’s data (including the **avatar**)